src/installer/publish/Directory.Build.targets

```diff
@@ -15,6 +15,10 @@
       $(DownloadDirectory)IntermediateUnsignedArtifacts\Windows_x64\Shipping\a.nupkg
   -->
   <Target Name="FindDownloadedArtifacts">
+    <PropertyGroup>
```

Comment:
All this should be deleted once there are no experimental packages in dotnet/runtime. Add a comment about it?

Reply:
Why should it be deleted? The way that the installer publishing infrastructure is done is to publish just the ridless assets from one leg (instead of all legs) because all installer legs produce ridless assets. It publishes that plus libraries_allconfigurations assets.

Reply:
I thought that this is another place where we need to override the check for experimental packages to get them published.

Is this change needed because publishing in dotnet/runtimelab is done differently (in a simpler way) than in dotnet/runtime?

Reply:
Yeah... this is because `dotnet/runtimelab` just publishes the packages it needs and we don't need to filter anything out.

Reply:
Hmm, this didn't work. I think I need to adjust how we upload packages instead.